tests/models/opt/test_modeling_opt.py

```diff
@@ -269,16 +269,28 @@ def _long_tensor(tok_lst):
 class OPTModelIntegrationTests(unittest.TestCase):
     @slow
     def test_inference_no_head(self):
+        # model is not deterministic on GPU, not sure why
```

Comment:
(some) non-deterministic is mentioned
https://pytorch.org/docs/stable/notes/randomness.html

The diff `1e-2` is still unpleasantly large though.

Reply:
Actually my comment is not great - the model is deterministic on GPU, but it's just that results differ depending on the CUDA vesion and CUDA != CPU

Reply:
@NielsRogge has this problem too before, and we end up with 2 expected values, one for CPU and one for GPU.
But now you mentioned different CUDA versions also count, so that approach is not super scalable.